scripts/log-capture

```diff
@@ -60,14 +60,21 @@ _to_log ip a
 _to_log ip r
 _to_log journalctl
 
+if [[ -e /sys/firmware/efi ]]; then
+    _to_log ls -l /sys/firmware/efi/efivars
+fi
+
 cp /etc/resolv.conf ${OUTDIR}
-cp /tmp/*.log ${OUTDIR}
-cp /root/lorax-packages.log ${OUTDIR}
+cp /tmp/*.log ${OUTDIR} 2>/dev/null
+
+if [[ -e /root/lorax-packages.log ]];then
+    cp /root/lorax-packages.log ${OUTDIR}
+fi
 
 if [[ -e /tmp/pre-anaconda-logs/ ]];then
     cp -r /tmp/pre-anaconda-logs ${OUTDIR}
 fi
 
-tar cfa ${ARCHIVE} ${OUTDIR}
+tar cfa ${ARCHIVE} ${OUTDIR} 2>/dev/null
```

Comment:
Compressing fails sometimes? I am not very happy about silencing the error messages. Can you please add more info on the reasoning for this, if you wish to keep as is?

Reply:
I added this to suppress "`Removing leading / from names`"

Reply:
Reworked with `tar -C` to avoid needing to suppress the message